test/specs/validate-address.spec.js

```diff
@@ -237,7 +237,7 @@ describe("validateAddress()", () => {
       street: ["170 Princes Blvd"],
       cityLocality: "Toronto",
       stateProvince: "On",
-      postalCode: "M6 K 3 C3",
+      postalCode: "M6K 3C3",
```

Comment:
I think this was a mistake in the previous implementation? This seems to be the expected formatting when reviewing addresses in Canada.

Reply:
Correct. Definitely a bug/limitation of my implementation.  

How does your implementation work?  Does it handle different postal code formats in different countries?

Reply:
Currently, it only capitalizes the postal code entry. I'm a little worried about the amount of normalization Simengine does for addresses. I could see this becoming quite complicated if we do it ourselves and it will likely still differ from the actual endpoint. 

Should we be farming this out to an external service? Maybe we expose something in the Platform just for normalization?

Reply:
I think your implementation is good enough, at least for now, possible forever.

I don't think it makes sense to create an external service for this. This sort of simulated logic (whether it's normalization, validation, mock business logic, etc.) is exactly what SimEngine is meant to do. The address validation stuff is probably the simplest example of it. Other endpoints will have _much_ more complicated simulation requirements.